docs/configuration.md

```diff
@@ -48,6 +48,7 @@ commit-date-format: 'yyyy-MM-dd'
 ignore:
   sha: []
   commits-before: yyyy-MM-ddTHH:mm:ss
+merge-messages: {}
```

Comment:
Wouldn't it be more consistent with existing config to name this `merge-message-formats`?

Reply:
Yes it would!  I'll get that fixed.

Reply:
Changed `merge-message` -> `merge-message-formats`